test/k8sT/manifests/graceful-termination.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: graceful-term-svc
+spec:
+  ports:
+    - port: 8081
+  selector:
+    app: graceful-term-server
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: graceful-term-server
+  labels:
+    app: graceful-term-server
+spec:
+  containers:
+    - name: server
```

Comment:
Do we actually need new pods to test the graceful termination? Maybe the existing testDS pods are enough? If not, then could you move the images into the cilium repo/org?

Reply:
Yes, we do - 1) Test phases are organized based on logged events by server/client 2) The server pod performs actions based on `SIGTERM` signal.
Yes, I've moved the image. Was waiting until Jenkins test infra issue was resolved.